oak/server/oak_runtime.cc

```diff
@@ -68,81 +112,36 @@ asylo::Status OakRuntime::Start() {
   LOG(INFO) << "Starting runtime";
 
   // Start the gRPC pseudo-node.
-  grpc_node_->Start();
+  if (grpc_node_ != nullptr) {
+    grpc_node_->Start();
+  }
 
   // Start the logging pseudo-node thread.
-  logging_node_->Start();
+  if (logging_node_ != nullptr) {
+    logging_node_->Start();
+  }
 
   // Start a new thread to process storage requests.
-  storage_node_->Start();
+  if (storage_node_ != nullptr) {
+    storage_node_->Start();
+  }
 
   // Now all dependencies are running, start the thread for the Node itself.
```

Comment:
Reword this to account for multiple nodes.

Reply:
Done.